google/resource_sql_database_instance.go

```diff
@@ -630,6 +634,11 @@ func resourceSqlDatabaseInstanceRead(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
+	// In the import case name won't be set yet
```

Comment:
I've had to add a `d.setId` to the Create as the resource relied on it being set at the end of the first Read.